ui/src/table.tsx

```diff
@@ -459,15 +462,16 @@ const
         wave.args[m.name] = [item.key as S]
         wave.push()
       },
-      onRenderItemColumn = (item?: Fluent.IObjectWithKey & Dict<any>, _idx?: U, col?: WaveColumn) => {
+      getItemColumnStyle = (item?: Fluent.IObjectWithKey & Dict<any>, _idx?: U, col?: WaveColumn) => {
         if (!item || !col) return <span />
```

Comment:
We know that both `item` and `col` are truthy at this point thanks to a check in `onRenderItemColumn`. No need for an additional check.

Nit: Naming could be better since we are not returning the style but component.

```suggestion
      getCellComponent = (item: Fluent.IObjectWithKey & Dict<any>,col: WaveColumn,  _idx?: U) => {
```

Reply:
Thanks for this suggestion, I removed the undefined check for `item` and `col` from `onRenderItemColumn` instead of removing it from `getCellComponent`. If I remove it from `getCellComponent` I would have to keep doing null checks for both `item` and `col` throughout `getCellComponent`, how does that sound?

Reply:
Sounds good.

> I would have to keep doing null checks for both item and col throughout getCellComponent

Not necessarily. You would just need to remove optional types in `getCellComponent` and make them required (since you already checked them in `onRenderItemColumn`). However, leaving them optional and checking just in `getCellComponent` is fine too. Thanks!